### PR TITLE
Updating and simplifying README

### PR DESCRIPTION
**Pull request by barndawgie**
I've significantly reworked the README, adding some links and updating to cover the recent changes to use config packages.

### CODE REVIEW

File: .github/workflows/fork_sync
Comment (tronikos) [inline .github/workflows/fork_sync:1]: Did you intentionally include this file? If yes, can you extract it to a different PR?

File: README.md [hunk 1]
```diff
@@ -35,92 +39,49 @@ Yellow  GND 5  ---       +--+
...
+### Step 1: Customizing the esphome-econet yaml for Your Device
 
-#### Heat Pump Water Heater
+Before installing, you will need to create a customized yaml for your device/environment. Start with either the [example-esp32.yaml](https://github.com/stockmopar/esphome-econet/blob/main/example-esp32.yaml) or [example-esp8266.yaml](https://github.com/stockmopar/esphome-econet/blob/main/example-esp8266.yaml) files as a template, depending on your chosen hardware.
 
-- `econet_heatpump_water_heater.yaml`
-
-#### HVAC
+At a minimum, you will need to either [put in your own WiFi network details](https://esphome.io/components/wifi) or provide a `secrets.yaml` file with the `wifi_ssid` and `wifi_password` fields configured. An example `secret.yaml` would look like:
+      ```yaml
```
Comment (tronikos) [inline README.md:56]: Can you make sure this renders correctly?
GitHub preview shows it as a single line https://github.com/stockmopar/esphome-econet/blob/54ff06667170875f0a90986b0bab50855e7250d3/README.md

File: README.md [hunk 2]
```diff
@@ -35,92 +39,49 @@ Yellow  GND 5  ---       +--+
...
-| WTR_BTUS      |               | Y            |              |          |            |
-| IGNCYCLS      |               | Y            |              |          |            |
-| BURNTIME      |               | Y            |              |          |            |
-
-Credits:
-
-- <https://github.com/syssi/esphome-solax-x1-mini>
-- <https://github.com/glmnet/esphome_devices>
-- <https://github.com/esphome/esphome/tree/dev/esphome/components/tuya>
+Open Home Assistant and add a new ESPHome Integration, inputing the hostname of your device (e.g. "econet-hpwh.local" by default for Heat Pump Water Heaters).  The device may be discovered automatically, in which case just accept the new device and wait a few moments.  A number of sensors should soon appear in `Developer Tools > States`.
```
Comment (tronikos) [inline README.md:87]: Remove the mention of dev tools? It's easier to find all sensors in the device itself.

**Review by tronikos — CHANGES_REQUESTED** [review]

File: .github/workflows/fork_sync
Comment (barndawgie) [reply .github/workflows/fork_sync:1]: Good catch. Reverted this.

File: README.md (hunk 1 above)
Comment (barndawgie) [reply README.md]: Need to add a newline. Fixed.

**Review by stockmopar — APPROVED** [review]
